$NetBSD: patch-doc_ref_api-compound.texi,v 1.1 2019/12/24 01:42:53 gutteridge Exp $

Remove an unnecessary non-breaking space character (code point 160)
that caused GNU texinfo 6.7 to fail during parsing.

--- doc/ref/api-compound.texi.orig	2010-12-13 17:24:39.000000000 +0000
+++ doc/ref/api-compound.texi
@@ -905,7 +905,7 @@ Sets the element at position @var{idx} i
 @end deftypefn
 
 @deftypefn {C Function} {const SCM *} scm_vector_elements (SCM vec, scm_t_array_handle *handle, size_t *lenp, ssize_t *incp)
-Acquire a handle for the vector @var{vec} and return a pointer to the
+Acquire a handle for the vector @var{vec} and return a pointer to the
 elements of it.  This pointer can only be used to read the elements of
 @var{vec}.  When @var{vec} is not a vector, an error is signaled.  The
 handle mustr eventually be released with
